ONSdigital/eq-survey-runner
# eq-285 multiple choice single answer

### Changes

To include radio boxes into questionnaire
### How to test
1. Run the questionnaire and select star wars from /dev. 
2. Make sure everything works and the radio boxes are mandatory on the 1st question and not the 2nd
3. Make sure all other questionnaires work
### Who can test

Anyone but @LJBabbage or @hamishtaplin
